i18n/ita/package.i18n.json

```diff
@@ -36,5 +36,13 @@
 	"pacCLI.envAndSolutionsPanel.copyVersionNumber.title": "Copia numero versione",
 	"microsoft-powerapps-portals.preview-show.title": "Portale PowerApps -> Mostra anteprima",
 	"microsoft-powerapps-portals.webExtension.init.title": "Inizializza estensione Web",
-	"microsoft-powerapps-portals.webExtension.limitation": "PAC CLI non è supportata in VS Code per il Web"
+	"microsoft-powerapps-portals.webExtension.limitation": "PAC CLI non è supportata in VS Code per il Web",
+	"microsoft-powerapps-portals.walkthrough.title": "Modifica codice Power Pages",
+	"microsoft-powerapps-portals.walkthrough.description": "Scopri come puoi creare siti Web completi e moderni direttamente in VS Code",
+	"microsoft-powerapps-portals.walkthrough.overview.title": "Panoramica",
+	"microsoft-powerapps-portals.walkthrough.overview.description": "L'estensione Power Pages abilita un file system personalizzato che viene sincronizzato con la struttura di progetto dello studio di Power Pages. \n \n L'anteprima è collegata a Power Pages e consente di visualizzare le modifiche.\n[Altre informazioni](command:powerplatform-walkthrough.overview-learn-more)",
+	"microsoft-powerapps-portals.walkthrough.fileSystem.title": "File system",
+	"microsoft-powerapps-portals.walkthrough.fileSystem.description": "I file vengono archiviati in cartelle annidate sotto il nome del sito. \n \n I singoli file di pagine sono disponibili nella cartella delle pagine Web sotto il nome del sito. \n \n Tutte le pagine sono organizzate in file HTML, CSS e JS all'interno della cartella con il rispettivo nome di pagina. \n \n Per altre informazioni sul file system Power Pages in VS Code per il Web, visita la nostra [documentazione](command:powerplatform-walkthrough.fileSystem-documentation). \n [Apri cartella del sito] (command:powerplatform-walkthrough.fileSystem-open-folder)",
```

Comment:
No spaces permitted between the `]` and `(` in the markdown link format.   
(2 in this file)